_attributes/common-attributes.adoc

```diff
@@ -170,4 +170,4 @@ endif::[]
 //Operator SDK version
 :osdk_ver: 1.25.4
 //Operator SDK version that shipped with the previous OCP 4.x release
-:osdk_ver_n1: 1.22.0
+:osdk_ver_n1: 1.22.2
```

Comment:
4.12 ships with osdk 1.25.4 currently: https://github.com/openshift/ocp-release-operator-sdk/blob/e11bcad257328713bfaa1f42160dc4d8e396cfa7/patches/03-setversion.patch#L10

```suggestion
:osdk_ver_n1: 1.25.4
```

Reply:
@everettraven: The attribute `osdk_ver_n1` is the version of the SDK that ships with the previous release, in this case 4.11. We use this attribute a couple of times in the upgrade docs. For example, the following markup in the [release notes:](https://58289--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-operator-sdk-1-25-z)
`If you have Operator projects that were previously created or maintained with Operator SDK {osdk_ver_n1}, update your projects to keep compatibility with Operator SDK {osdk_ver}.`

Renders as:
If you have Operator projects that were previously created or maintained with Operator SDK 1.22.2, update your projects to keep compatibility with Operator SDK 1.25.4.

Reply:
Ahhhh gotcha. That makes sense - thanks for the clarification!

Reply:
/resolve